python/pyxet/pyxet/cli.py

```diff
@@ -125,7 +125,7 @@ def cp(source: Annotated[str, typer.Argument(help="Source file or folder to copy
                int, typer.Option("--parallel", "-p", help="Maximum amount of parallelism")] = 32):
         """copy files and folders"""
         if not message:
-            message = f"copy {source} to {target}" if not recursive else f"copy {source} to {target} recursively"
+            message = f"copy {source[0]}... to {target}" if not recursive else f"copy {source[0]}... to {target} recursively"
```

Comment:
Will crash on empty list...  
Let's actually do `{source[:3].join(', ')}...`

Reply:
Added a check for empty source list.